libs/util/dsg-utils/src/logger/logging.ts

```diff
@@ -0,0 +1,16 @@
+import { Logger, LogLevel } from "@amplication/util-logging";
+
+//const { LOG_LEVEL, NODE_ENV } = process.env;
+
+const LOG_LEVEL = process.env["LOG_LEVEL"] || "INFO";
+const NODE_ENV = process.env["NODE_ENV"] || "Development";
```

Comment:
Note that you need the matching PR in cluster-configuration to add the NODE_ENV to the DSG:

https://github.com/amplication/amplication-cluster-configuration/blob/main/production/applications/data-service-generator/values.yaml